bin/appliance_console

```diff
@@ -393,6 +393,14 @@ Static Network Configuration
 
         messaging_action = ask_with_menu("Configure Messaging", messaging_options)
 
+        no_changes = database_action == "no_changes" && messaging_action == "no_changes"
```

Comment:
Perhaps, instead we make it a positive variable?

```suggestion
        changes_requested = database_action != "no_changes" || messaging_action != "no_changes"
```

Reply:
Or

```suggestion
        changes_requested = !(database_action == "no_changes" && messaging_action == "no_changes")
```

Reply:
:+1: done